target/linux/ramips/dts/RT-AC54U.dts

```diff
@@ -0,0 +1,152 @@
+/dts-v1/;
```

Comment:
Please check the ["submitting patches guidelines"](https://openwrt.org/submitting-patches#dts_checklist). You need to add SPDX-License-Identifier at the top of this file.

Reply:
OK